Assets/Scripts/TerrainDestroyer.cs

```diff
@@ -29,33 +29,16 @@ private void Awake()
 
     }
 
-    // Caculates the distance from the explosion to each potential from that center
-    // It then destroys any tile that is within the the radius of the explosion
-    public void DestroyTerrain(Vector3 explosionLocation, float radius)
-    {
-
-        /// The step function moves up by a small step because that is the scale of the tile map and we need to check every tile for this to work
-        for (float x = -radius; x < radius; x += 0.05f)
-        {
-            for (float y = -radius; y < radius; y += 0.05f)
-            {
-
-                if (Mathf.Pow(x, 2) + Mathf.Pow(y, 2) < Mathf.Pow(radius, 2))
-                {
-                    Vector3 translatedPosition = explosionLocation + new Vector3(x, y, 0);
-                    Vector3Int tilePos = terrain.WorldToCell(translatedPosition);
-                    if (terrain.GetTile(tilePos) != null)
-                    {
-                        DestroyTile(tilePos);
-                    }
-                }
-            }
+    // Destroys terrain, called from the Bomb with specific position
+    public void DestroyTerrain_Bomb(Vector3 translatedPosition) {
+        Vector3Int tilePos = terrain.WorldToCell(translatedPosition);
+        if (terrain.GetTile(tilePos) != null) {
+            DestroyTile(tilePos);
         }
-
     }
 
     // Sets the tile in the foreground to nil and draws the background tile.
-    void DestroyTile(Vector3Int tilePosition)
+    public void DestroyTile(Vector3Int tilePosition)
```

Comment:
Could this stay non-public?

Reply:
yeah, it could.  Sorry, that was a remnant from something I was testing earlier and forgot to change it back